### ISSUE TITLE
False positive for JSON security hardening when using named argument for flags

### ISSUE THREAD

**Issue by sandacode** [kalessil/phpinspectionsea#1972]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended)  |
| Language level | 8.3                        |

Current behaviour
--
```
json_decode('[]', flags: JSON_THROW_ON_ERROR);
>> [EA] Please consider taking advantage of JSON_THROW_ON_ERROR flag for this call options. 

Expected behaviour
--
json_decode('[]', flags: JSON_THROW_ON_ERROR);
>> EA does not report an issue

Environment details
---
PHPStorm 2024.3.5
Build #PS-243.26053.13, built on March 12, 2025
MacOS 15.3.2 (24D81) / Apple Silicon
Runtime version: 21.0.6+8-b631.39 aarch64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.

**Comment by jdreesen** [comment]
Duplicate of #1925, #1873, #1868, #1845, #1825, #1807, #1725, #1639, …

**Comment by sander-bol** [comment]
D'oh. Sorry about that!

**Comment by ea-inspections-team** [comment]
Thank you for reporting @sandacode  ! Fixed in #1977.
